### PR TITLE
Set up development environment

### PR DESCRIPTION
**Pull request by ecleary**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
-# reservation
\ No newline at end of file
+# Project Name
+
+> Project description
+
+## Related Projects
+
+  - https://github.com/teamName/repo
```
Comment (tdomingo) [inline README.md]: be sure to provide links to our other components here when the time is right 👍

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<html>
+  <head>
+    <meta charset="utf8">
+    <title>ZenRent Reservation</title>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script type="text/javascript" src="./app.js"></script>
+  </body>
+</html>
```
Comment (tdomingo) [inline client/public/index.html]: add ending newline

**Review by tdomingo — APPROVED** [review]
Looks great! 

I'd consider possibly including server and database directories in your repo in the init, as well as serving your bundled app.js, but that's definitely your call 👍

File: README.md (hunk 1 above)
Comment (ecleary) [reply README.md]: Fixed. Thanks!

File: client/public/index.html (hunk 2 above)
Comment (ecleary) [reply client/public/index.html]: Fixed. Thanks!

**Comment by ecleary** [comment]
Thanks @tdomingo, actually those directories do exist on my local, but seems that git doesn't stage or commit them if they don't contain any files. Not sure if you've found a way around that? Regardless, they won't be empty for long.

Other fixes made, so will proceed with merging into main.

**Pull request merged**